lib/ansible/galaxy/api.py

```diff
@@ -46,6 +53,15 @@ def wrapped(*args, **kwargs):
     return wrapped
 
 
+def rate_limit_exception(exception=None):
+    # Note: cloud.redhat.com masks rate limit errors with 403 (Forbidden) error codes.
+    # Since 403 could reflect the actual problem (such as an expired token), we should
+    # not retry by default.
+    if isinstance(exception, GalaxyError) and exception.http_code in RETRY_HTTP_ERROR_CODES:
+        return True
+    return False
```

Comment:
The if-block seems a litte excessive, could just do
```suggestion
    return isinstance(exception, GalaxyError) and exception.http_code in RETRY_HTTP_ERROR_CODES
```